library/spdm_requester_lib/libspdm_req_set_certificate.c

```diff
@@ -87,11 +95,33 @@ static libspdm_return_t libspdm_try_set_certificate(libspdm_context_t *spdm_cont
     spdm_request->header.param1 = slot_id;
     spdm_request->header.param2 = 0;
 
+    if (spdm_request->header.spdm_version >= SPDM_MESSAGE_VERSION_13) {
+        if ((request_attribute & SPDM_GET_CERTIFICATE_REQUEST_SLOT_ID_MASK) != 0) {
```

Comment:
Because there are two input parameters for slot_id:  `slot_id` and `request_attribute`.  To ensure user don't put two different slot_id, the API added the comment: 
```
 * @param  request_attribute            Set certificate request attributes. This field is only used for SPDM 1.3 and above.
 *                                      And the bit[0~3] of request_attribute must be 0.
```